<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Yaminis-MacBook-Air.local" failures="2" tests="38" name="com.tekarch.tests.AutomationScripts" time="913.807" errors="0" timestamp="2022-05-19T00:59:42 PDT" skipped="0">
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase1" time="16.688"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase2" time="16.338"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase3" time="25.581"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase4a" time="16.019"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase4b" time="17.590"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase5" time="16.739"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase6" time="70.512"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase7" time="33.735"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase8" time="20.711"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase9" time="20.332"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase10" time="22.353"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase11" time="23.167"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase12" time="23.463"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase13" time="22.639">
    <failure message="no such alert
  (Session info: chrome=101.0.4951.64)
Build info: version: &#039;4.1.4&#039;, revision: &#039;535d840ee2&#039;
System info: host: &#039;Yaminis-MacBook-Air.local&#039;, ip: &#039;2600:8802:180c:8c00:dcfb:454a:9884:b63%en0&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;12.1&#039;, java.version: &#039;11.0.14&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [ca5699126570aeff4c33ed8ace803bcd, getAlertText {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 101.0.4951.64, chrome: {chromedriverVersion: 101.0.4951.41 (93c720db8323..., userDataDir: /var/folders/fc/qxv5mypn205...}, goog:chromeOptions: {debuggerAddress: localhost:52526}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), se:cdp: ws://localhost:52526/devtoo..., se:cdpVersion: 101.0.4951.64, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: ca5699126570aeff4c33ed8ace803bcd expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: no such alert
  (Session info: chrome=101.0.4951.64)
Build info: version: '4.1.4', revision: '535d840ee2'
System info: host: 'Yaminis-MacBook-Air.local', ip: '2600:8802:180c:8c00:dcfb:454a:9884:b63%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '12.1', java.version: '11.0.14'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [ca5699126570aeff4c33ed8ace803bcd, getAlertText {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 101.0.4951.64, chrome: {chromedriverVersion: 101.0.4951.41 (93c720db8323..., userDataDir: /var/folders/fc/qxv5mypn205...}, goog:chromeOptions: {debuggerAddress: localhost:52526}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), se:cdp: ws://localhost:52526/devtoo..., se:cdpVersion: 101.0.4951.64, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: ca5699126570aeff4c33ed8ace803bcd expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at com.tekarch.tests.AutomationScripts.testCase13(AutomationScripts.java:669)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testCase13 -->
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase14" time="32.651"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase15" time="18.594"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase16" time="25.495"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase17" time="20.707"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase18" time="23.464"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase19" time="15.597"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase20" time="23.527"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase21" time="18.468"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase22" time="33.373"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase23" time="12.248"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase24" time="24.792"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase25" time="21.746"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase26" time="29.171"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase27" time="17.957"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase28" time="25.676"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase29" time="18.631"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase30" time="25.470"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase31" time="23.331"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase32" time="20.050">
    <failure message="New Contact Creation Failed.Error: Invalid Data.
Review all error messages below to correct your data. expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: New Contact Creation Failed.Error: Invalid Data.
Review all error messages below to correct your data. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at com.tekarch.tests.AutomationScripts.testCase32(AutomationScripts.java:1469)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testCase32 -->
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase33" time="19.365"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase34" time="28.856"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase35" time="32.164"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase36" time="31.593"/>
  <system-out/>
  <testcase classname="com.tekarch.tests.AutomationScripts" name="testCase37" time="25.014"/>
  <system-out/>
</testsuite> <!-- com.tekarch.tests.AutomationScripts -->
